dtrace_probe.cc

```diff
@@ -59,8 +61,17 @@ namespace node {
       return Nan::Undefined();
     }
 
+    int cblen = argsinfo.Length() - fnidx - 1;
+    Local<Value> *cbargs = new Local<Value>[cblen];
```

Comment:
Is it possible to use an array initialization literal and allocate this on the stack instead of using `new/delete`?